### PR TITLE
Adds some documentation

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

File: mcc/capcom/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Installation
+
+    go get github.com/poka-yoke/spaceflight/mcc/capcom
+
+## Usage
+
+    cacpom list
+    capcom add --source 198.234.12.34 sg-459d024
+    capcom revoke --source 198.234.12.34 sg-459d024
+
```
Comment (miguelbernadi) [inline mcc/capcom/README.md]: Would be nice to mention it has in-tool help as well.

File: mcc/got/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Installation
+
+    go get github.com/poka-yoke/spaceflight/mcc/got
+
+## Usage
+
+    got ttl --zone example.com -ttl 30
+    got upsert --name www.example.com. --zone example.com --ttl 300 --type CNAME myserver.example.com
+    got ttl --zone example.com -ttl 360
+
```
Comment (miguelbernadi) [inline mcc/got/README.md]: Would be nice to mention it has in-tool help as well.

File: mcc/trek/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# trek
+
+## Description
+
+`trek` is a redirection generation tool.
```
Comment (miguelbernadi) [inline mcc/trek/README.md:5]: for nginx configuration files.

**Comment by miguelbernadi** [comment]
+1, some minor nitpicking.

File: mcc/capcom/README.md (hunk 1 above)
Comment (ifosch) [reply mcc/capcom/README.md]: +1

File: mcc/got/README.md (hunk 2 above)
Comment (ifosch) [reply mcc/got/README.md]: +1

File: mcc/trek/README.md (hunk 3 above)
Comment (ifosch) [reply mcc/trek/README.md:5]: +1

**Comment by ifosch** [comment]
~~I'll amend these later, thanks!~~ Done @miguelbernadi

**Pull request merged**